internal/cmd/primaryip/create.go

```diff
@@ -34,31 +33,47 @@ var CreateCmd = base.Cmd{
 		cmd.RegisterFlagCompletionFunc("datacenter", cmpl.SuggestCandidatesF(client.Datacenter().Names))
 
 		cmd.Flags().StringToString("label", nil, "User-defined labels ('key=value') (can be specified multiple times)")
+
+		cmd.Flags().StringSlice("enable-protection", []string{}, "Enable protection (delete, rebuild) (default: none)")
+		cmd.RegisterFlagCompletionFunc("enable-protection", cmpl.SuggestCandidates("delete", "rebuild"))
```

Comment:
Only [`delete`](https://docs.hetzner.cloud/#primary-ip-actions-change-primary-ip-protection) is a valid protection level for primary ips.